### DISCUSSION TITLE
Is it possible to make a native module that imports a cocoapod?

### DISCUSSION THREAD

**Discussion by ggaabe** [callstack/react-native-builder-bob#548] [Q&A] [answered]
Hi, I'm needing to make a react-native native module that imports and re-exposes methods from an iOS cocoapods package. I then need to publish this module as a react-native npm package, and also do it in such a way that it will contain the needed pod dependencies.

Are there any guides around how to do this?

**Comment by satya164** [comment, marked as answer]
You'll need to specify it in the podspec with `s.dependency`:

```js
s.dependency 'Your-Pod-Name', '1.2.3'
```

**Comment by ggaabe** [comment]
@satya164 Thanks; I have one additional question; it seems the iOS build for the example project doesn't work. I get this error:

```
/Users/gabrielgarrett/Library/Developer/Xcode/DerivedData/SdkExample-gsywxmgudbutecahyvqzbyvxlrre/Build/Intermediates.noindex/SdkExample.build/Debug-iphoneos/SdkExample.build/DerivedSources/SdkExample_vers.c module map file '/Users/gabrielgarrett/Library/Developer/Xcode/DerivedData/SdkExample-gsywxmgudbutecahyvqzbyvxlrre/Build/Products/Debug-iphoneos/react-native-sdk/react_native_sdk.modulemap' not found

```

Has this been seen before? This occurred before I even tried installing any additional pods
